src/pages/Anime365/tests.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Anime365",
+  "url": "https://smotret-anime.com",
+  "testCases": [
+    {
+      "url": "https://smotret-anime.com/catalog/genres",
```

Comment:
Use an anime overview page. sync false in this case mean that it is a overview page